regulations/templates/regulations/preamble-partial.html

```diff
@@ -32,10 +32,12 @@
 
 <script id="comment-index-template" type="text/template">
   <% _.each(comments, function(comment) { %>
-    <div class="comment-index-item" data-comment-section="<%= comment.section %>">
-      <span><%= comment.section %></span>
-      <span class="comment-index-edit">Edit</span>
-      <span class="comment-index-clear">Clear</span>
-    </div>
+    <li class="comment-index-item group" data-comment-section="<%= comment.section %>">
+      <div class="comment-index-section"><%= comment.section %></div>
+      <div class="comment-index-modify">
+        <span class="comment-index-edit">edit</span>
+        <span class="comment-index-clear">x</span>
```

Comment:
Probably a good idea to replace this with an icon (there's one in our font for "delete", I think) along with the "close" text for accessibility.